{
  "mitigation": "Mitigation for this issue is either not available or the currently available options do not meet the Red Hat Product Security criteria comprising ease of use and deployment, applicability to widespread installation base or stability.",
  "affected_release": null,
  "package_state": [
    {
      "product_name": "OpenShift Lightspeed",
      "fix_state": "Fix deferred",
      "package_name": "openshift-lightspeed-tech-preview/lightspeed-service-api-rhel9",
      "cpe": "cpe:/a:redhat:openshift_lightspeed"
    },
    {
      "product_name": "Red Hat Enterprise Linux AI (RHEL AI)",
      "fix_state": "Not affected",
      "package_name": "rhelai1/bootc-amd-rhel9",
      "cpe": "cpe:/a:redhat:enterprise_linux_ai:1"
    },
    {
      "product_name": "Red Hat Enterprise Linux AI (RHEL AI)",
      "fix_state": "Not affected",
      "package_name": "rhelai1/bootc-aws-nvidia-rhel9",
      "cpe": "cpe:/a:redhat:enterprise_linux_ai:1"
    },
    {
      "product_name": "Red Hat Enterprise Linux AI (RHEL AI)",
      "fix_state": "Not affected",
      "package_name": "rhelai1/bootc-azure-amd-rhel9",
      "cpe": "cpe:/a:redhat:enterprise_linux_ai:1"
    },
    {
      "product_name": "Red Hat Enterprise Linux AI (RHEL AI)",
      "fix_state": "Not affected",
      "package_name": "rhelai1/bootc-azure-nvidia-rhel9",
      "cpe": "cpe:/a:redhat:enterprise_linux_ai:1"
    },
    {
      "product_name": "Red Hat Enterprise Linux AI (RHEL AI)",
      "fix_state": "Not affected",
      "package_name": "rhelai1/bootc-gcp-nvidia-rhel9",
      "cpe": "cpe:/a:redhat:enterprise_linux_ai:1"
    },
    {
      "product_name": "Red Hat Enterprise Linux AI (RHEL AI)",
      "fix_state": "Not affected",
      "package_name": "rhelai1/bootc-ibm-nvidia-rhel9",
      "cpe": "cpe:/a:redhat:enterprise_linux_ai:1"
    },
    {
      "product_name": "Red Hat Enterprise Linux AI (RHEL AI)",
      "fix_state": "Not affected",
      "package_name": "rhelai1/bootc-intel-rhel9",
      "cpe": "cpe:/a:redhat:enterprise_linux_ai:1"
    },
    {
      "product_name": "Red Hat Enterprise Linux AI (RHEL AI)",
      "fix_state": "Not affected",
      "package_name": "rhelai1/bootc-nvidia-rhel9",
      "cpe": "cpe:/a:redhat:enterprise_linux_ai:1"
    },
    {
      "product_name": "Red Hat Enterprise Linux AI (RHEL AI)",
      "fix_state": "Not affected",
      "package_name": "rhelai1/docling-serve-rhel9",
      "cpe": "cpe:/a:redhat:enterprise_linux_ai:1"
    },
    {
      "product_name": "Red Hat Enterprise Linux AI (RHEL AI)",
      "fix_state": "Not affected",
      "package_name": "rhelai1/instructlab-amd-rhel9",
      "cpe": "cpe:/a:redhat:enterprise_linux_ai:1"
    },
    {
      "product_name": "Red Hat Enterprise Linux AI (RHEL AI)",
      "fix_state": "Not affected",
      "package_name": "rhelai1/instructlab-intel-rhel9",
      "cpe": "cpe:/a:redhat:enterprise_linux_ai:1"
    },
    {
      "product_name": "Red Hat Enterprise Linux AI (RHEL AI)",
      "fix_state": "Not affected",
      "package_name": "rhelai1/instructlab-nvidia-rhel9",
      "cpe": "cpe:/a:redhat:enterprise_linux_ai:1"
    },
    {
      "product_name": "Red Hat Enterprise Linux AI (RHEL AI)",
      "fix_state": "Not affected",
      "package_name": "rhelai1/ui-rhel9",
      "cpe": "cpe:/a:redhat:enterprise_linux_ai:1"
    },
    {
      "product_name": "Red Hat OpenShift AI (RHOAI)",
      "fix_state": "Not affected",
      "package_name": "rhoai/odh-ml-pipelines-api-server-v2-rhel8",
      "cpe": "cpe:/a:redhat:openshift_ai"
    },
    {
      "product_name": "Red Hat OpenShift AI (RHOAI)",
      "fix_state": "Not affected",
      "package_name": "rhoai/odh-ml-pipelines-driver-rhel8",
      "cpe": "cpe:/a:redhat:openshift_ai"
    },
    {
      "product_name": "Red Hat OpenShift AI (RHOAI)",
      "fix_state": "Not affected",
      "package_name": "rhoai/odh-ml-pipelines-launcher-rhel8",
      "cpe": "cpe:/a:redhat:openshift_ai"
    },
    {
      "product_name": "Red Hat OpenShift AI (RHOAI)",
      "fix_state": "Not affected",
      "package_name": "rhoai/odh-ml-pipelines-persistenceagent-v2-rhel8",
      "cpe": "cpe:/a:redhat:openshift_ai"
    },
    {
      "product_name": "Red Hat OpenShift AI (RHOAI)",
      "fix_state": "Not affected",
      "package_name": "rhoai/odh-ml-pipelines-scheduledworkflow-v2-rhel8",
      "cpe": "cpe:/a:redhat:openshift_ai"
    }
  ],
  "threat_severity": "Low",
  "public_date": "2025-03-30T15:31:04Z",
  "bugzilla": {
    "description": "torch: PyTorch torch.mkldnn_max_pool2d denial of service",
    "id": "2356078",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2356078"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "3.3",
    "cvss3_scoring_vector": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:L",
    "status": "draft"
  },
  "iava": "",
  "cwe": "CWE-404",
  "statement": "",
  "acknowledgement": "",
  "name": "CVE-2025-2953",
  "document_distribution": "",
  "details": [
    "A vulnerability, which was classified as problematic, has been found in PyTorch 2.6.0+cu124. Affected by this issue is the function torch.mkldnn_max_pool2d. The manipulation leads to denial of service. An attack has to be approached locally. The exploit has been disclosed to the public and may be used.",
    "A flaw was found in PyTorch. This vulnerability allows denial of service via local exploitation."
  ],
  "references": [
    "https://www.cve.org/CVERecord?id=CVE-2025-2953\nhttps://nvd.nist.gov/vuln/detail/CVE-2025-2953\nhttps://github.com/pytorch/pytorch/issues/149274\nhttps://github.com/pytorch/pytorch/issues/149274#issue-2923122269\nhttps://vuldb.com/?ctiid.302006\nhttps://vuldb.com/?id.302006\nhttps://vuldb.com/?submit.521279"
  ]
}